<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommentaryForm</class>
 <widget class="QWidget" name="CommentaryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="toolButton_backward">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../icons.qrc">
           <normaloff>:/icons/16x16/go-previous.png</normaloff>:/icons/16x16/go-previous.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_forward">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../icons.qrc">
           <normaloff>:/icons/16x16/go-next.png</normaloff>:/icons/16x16/go-next.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_openInBrowser">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../icons.qrc">
           <normaloff>:/icons/16x16/internet-web-browser.png</normaloff>:/icons/16x16/internet-web-browser.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_saveLocal">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../icons.qrc">
           <normaloff>:/icons/16x16/download.png</normaloff>:/icons/16x16/download.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
